diff --git a/node_modules/rc-tree/lib/interface.d.ts b/node_modules/rc-tree/lib/interface.d.ts
index 594ca9c..3bbfb50 100644
--- a/node_modules/rc-tree/lib/interface.d.ts
+++ b/node_modules/rc-tree/lib/interface.d.ts
@@ -56,7 +56,7 @@ export type Key = React.Key;
  * https://github.com/microsoft/TypeScript/issues/50217
  */
 export type SafeKey = Exclude<Key, bigint>;
-export type KeyEntities<DateType = any> = Record<SafeKey, DataEntity<DateType>>;
+export type KeyEntities<DateType extends object = any> = Record<SafeKey, DataEntity<DateType>>;
 export type DataNode = FieldDataNode<{
     key: Key;
     title?: React.ReactNode | ((data: DataNode) => React.ReactNode);
